README.md

```diff
@@ -41,6 +40,8 @@ Installer:
 
 You can generate the installer with `ninja win-installer` after a successful build. This assumes a working internet connection and installation of the optional dependencies.
 
+You can generate the portable zip with `meson compile win-portable -C build` after a successful build.
```

Comment:
I'd change this to match the previous line and just say `ninja win-portable` is the recommended command. Meson itself is moving away from `meson compile` since in practice it hasn't proved as build system-agnostic as Jussi had hoped.